src/commands/syncModel/run/README.md

```diff
@@ -66,6 +66,35 @@ const params: SyncModelRunParams = {
 await syncModelRun(params);
 ```
 
+### Partial Sync
+
+The tool allows you to control which data from your environment gets synced. Using the CLI command, you can filter data by entity type with the `--include` or `--exclude` parameters. Additionally, you can specify an array of codenames to be synced using the `--codenames` parameter, which will apply the codename filter across all synced entities. For more advanced filtering, you can use the `filter` parameter through [programmatic sync](#sync-model-programmatically), where you can define custom filter predicate functions for each entity. These functions are optional, and if not provided, the entity will be synced by default.
```

Comment:
This is now different. Only the `entities` and `codenames` parameters are present, right?